tests/unit/models/DefaultPlugin.spec.js

```diff
@@ -0,0 +1,108 @@
+import DefaultPlugin from 'src/models/DefaultPlugin';
+
+describe('Test class: DefaultPlugin', () => {
+  describe('Test constructor', () => {
+    it('Check variable instantiation', () => {
+      const plugin = new DefaultPlugin();
+
+      expect(plugin.data).toBeNull();
+      expect(plugin.__drawer).toBeNull();
+      expect(plugin.__metadata).toBeNull();
+      expect(plugin.__parser).toBeNull();
+      expect(plugin.__renderer).toBeNull();
+    });
+
+    it('Check passing undefined variables to constructor', () => {
+      const plugin = new DefaultPlugin({});
+
+      expect(plugin.data).toBeNull();
+      expect(plugin.__drawer).toBeNull();
+      expect(plugin.__metadata).toBeNull();
+      expect(plugin.__parser).toBeNull();
+      expect(plugin.__renderer).toBeNull();
+    });
+
+    it('Check passing all variables to constructor', () => {
+      const plugin = new DefaultPlugin({
+        pluginData: 1,
+        pluginDrawer: 2,
+        pluginMetadata: 3,
+        pluginParser: 4,
+        pluginRenderer: 5,
+      });
+
+      expect(plugin.data).toEqual(1);
+      expect(plugin.__drawer).toEqual(2);
+      expect(plugin.__metadata).toEqual(3);
+      expect(plugin.__parser).toEqual(4);
+      expect(plugin.__renderer).toEqual(5);
+    });
+  });
```

Comment:
Unsure what the finality of these tests are, the all look like situations where plugin instantiation should fail *loudly* to me.

Reply:
To check if we set the good thing at the good place.